FlashCards.AhmadJer99/FlashCards/Data/CardsDBController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Dapper;
+using FlashCards.Models;
+
+namespace FlashCards.Data;
+
+internal class CardsDBController : DataBaseController<Card>
```

Comment:
🟠  **Naming Conventions**

💡 Use PascalCase.

🔧 `CardsDbController` or `CardsDatabaseController`. 

🗒️ Database is one word, so captialising the B as you did in `DataBase` is redundant and confusing. But database also commonly abbreviated to DB, which would then be treated as one word, so `Db`.

Reply:
Done fixed that!